benchmark/king2dcompressible/ala/README

```diff
@@ -0,0 +1,18 @@
+This folder contains input files for the King et al. 2010 benchmark
+"A community benchmark for 2-D Cartesian compressible convection
+in the Earth’s mantle". 
+This includes versions for the ALA and TALA approximations for 
+different values of the Dissipation number Di and the Rayleight 
+number Ra. The complete input file can be found in 'ala.prm'. 
+All other input files only contain the modifications necessary 
+ro run cases with a different approximation or values of Di and Ra, 
+and the files can be combined using cat, for example
+
+  cat ala.prm tala.prm ala_Di05_Ra5.prm | ./aspect --
+
+to run a model with the TALA approximation, Di=0.5 and Ra=10^5. 
+
+The bash script 'get_statistics' can copy the relevant statistics
+of one or several model runs to the screen output and a separate
+file. It takes one or several paths to a number of statistics 
+files as input. 
```

Comment:
That description is too vague for me to really understand what the script is doing.